src/main/java/com/github/fabienrenaud/jjb/provider/ClientsJsonProvider.java

```diff
@@ -144,6 +146,13 @@ public void toJson(com.squareup.moshi.JsonWriter writer, @Nullable OffsetDateTim
     private final DslJson<Object> dsljson = new DslJson<>(Settings.withRuntime().includeServiceLoader());
     private final DslJson<Object> dsljson_reflection = new DslJson<>(Settings.withRuntime());//don't include generated classes
 
+    private final io.avaje.jsonb.JsonType<Clients> avajeJsonb_jackson = io.avaje.jsonb.Jsonb
+            .newBuilder().serializeEmpty(true)
+            .adapter(new JacksonAdapter(true, true, false)).build().type(Clients.class);
```

Comment:
nit: would be nice to use comments to clarify the meaning of these parameters.
e.g: `new JacksonAdapter(/*useFoo=*/ true, /*disableBlah=*/=true, /*yolo=*/false)`;
ditto for other similar calls.

Reply:
Yes, just done that.